doc/dftb+/manual/dftbp.tex

```diff
@@ -3079,6 +3080,13 @@ \subsection{Casida}
   \item[\is{TestArnoldi}] If set to \is{Yes}, the file TEST\_ARPACK.DAT is created, which gives data
     on the quality of the resulting eigenstates.
 
+  \item[\is{ExcitedStateForces}] If ground state forces are enabled, either due
+    to a geometry driver (p.~\pref{sec:dftbp.Driver}) or requested in
+    \is{Analysis} (p.~\pref{sec:dftbp.Analysis}), this option includes the
+    contributions from the excited state of interest (if set to \is{Yes}). This
+    option can be used, for example, to calculate excitation energies during a
+    ground state molecular dynamics simulation.
+
```

Comment:
I find the example confusing, as users would ask: Why do I need this option to calculate excited states energies during MD? Why not: By setting this option to No, .... (Not setting the option explicitely to No during the molecular dynamics may prevent the calculation of the excited state spectrum for those cases, where the excited states can not be calcualted)

Or something similar...

Reply:
Not sure I follow the replacement text either. Have re-written the text, so please review .